### PR TITLE
Add some options

### PR DESCRIPTION
**Pull request by swflint**
This PR adds a couple of options and should help to make the script slightly more useful.

### CODE REVIEW

File: epipe [hunk 1]
```diff
@@ -10,6 +10,7 @@ usage() {
     echo "$self redirects stdin to an emacs buffer"
     echo -e "Example: 'echo \"foo\" | epipe'"
     echo
+    echo -e "\t-e CMD\t Use emacs CMD."
```
Comment (dajva) [inline epipe]: May possibly mention the env variable here as well.

**Review by dajva — APPROVED** [review]
Thanks a lot. I actually do think there are better alternative to this if you want true piping and not just auto revert on a temp file. Couldn't find it now after a quick search. Personally I do like the temp file solution despite its drawbacks (auto revert is not optimal).
Had one suggestion for the documentation but the patch is fine as is. So if you don't do anything about it I will merge in a few days.

File: epipe (hunk 1 above)
Comment (swflint) [reply epipe]: Have updated.

**Comment by swflint** [comment]
Hopefully that's better!

**Comment by dajva** [comment]
Thanks!

**Pull request merged**